source/colors.js

```diff
@@ -15,255 +15,354 @@ module.exports = {
+        { tone: '90', value: '#272149' }
       ]
     },
     {
       name: 'gray',
       values: [
         { tone: 'core', value: '#777677' },
-        { tone: '10', value: '#e0e0e0' },
-        { tone: '20', value: '#c7c7c7' },
-        { tone: '30', value: '#aeaeae' },
-        { tone: '40', value: '#959595' },
+        { tone: '1', value: '#eaeaea' },
+        { tone: '10', value: '#d8d8d8' },
+        { tone: '20', value: '#c0bfc0' },
+        { tone: '30', value: '#a6a5a6' },
+        { tone: '40', value: '#949394' },
         { tone: '50', value: '#777677' },
-        { tone: '60', value: '#5a5a5a' },
+        { tone: '60', value: '#595859' },
         { tone: '70', value: '#464646' },
-        { tone: '80', value: '#323232' },
-        { tone: '90', value: '#121212' },
-        { tone: '100', value: '#000' }
+        { tone: '80', value: '#343334' },
+        { tone: '90', value: '#272727' }
       ]
     },
     {
-      name: 'cool-gray',
+      name: 'cool gray',
       values: [
-        { tone: 'core', value: '#6d7777' },
-        { tone: '10', value: '#dfe9e9' },
-        { tone: '20', value: '#c8d2d2' },
-        { tone: '30', value: '#aeb8b8' },
-        { tone: '40', value: '#959f9f' },
-        { tone: '50', value: '#6d7777' },
-        { tone: '60', value: '#5a6464' },
-        { tone: '70', value: '#3c4646' },
-        { tone: '80', value: '#323c3c' },
-        { tone: '90', value: '#0d1111' },
-        { tone: '100', value: '#000203' }
+        { tone: 'core', value: '#6f7878' },
+        { tone: '1', value: '#e3ecec' },
+        { tone: '10', value: '#d0dada' },
+        { tone: '20', value: '#b8c1c1' },
+        { tone: '30', value: '#9fa7a7' },
+        { tone: '40', value: '#8c9696' },
+        { tone: '50', value: '#6f7878' },
+        { tone: '60', value: '#535a5a' },
+        { tone: '70', value: '#424747' },
+        { tone: '80', value: '#343334' },
+        { tone: '90', value: '#272727' }
       ]
     },
     {
-      name: 'warm-gray',
+      name: 'warm gray',
       values: [
         { tone: 'core', value: '#7d7373' },
-        { tone: '10', value: '#e9e0e0' },
-        { tone: '20', value: '#d0c7c7' },
-        { tone: '30', value: '#b8aeae' },
-        { tone: '40', value: '#9e9494' },
+        { tone: '1', value: '#efe9e9' },
+        { tone: '10', value: '#e2d5d5' },
+        { tone: '20', value: '#ccbcbc' },
+        { tone: '30', value: '#b4a1a1' },
+        { tone: '40', value: '#9e9191' },
         { tone: '50', value: '#7d7373' },
-        { tone: '60', value: '#645a5a' },
-        { tone: '70', value: '#504646' },
-        { tone: '80', value: '#3c3232' },
-        { tone: '90', value: '#1a1314' },
-        { tone: '100', value: '#030000' }
+        { tone: '60', value: '#5f5757' },
+        { tone: '70', value: '#4b4545' },
+        { tone: '80', value: '#373232' },
+        { tone: '90', value: '#2a2626' }
       ]
     },
     {
       name: 'neutral-white',
```

Comment:
De-hyphenate as in https://github.com/IBM-Design/colors/pull/27/files#diff-6a853a1f7639b0af7520dddefa30c2ffR292